library/pod-security-policy/apparmor/template.yaml

```diff
@@ -15,6 +15,7 @@ spec:
           type: object
           properties:
             allowedProfiles:
+              description: "An array of AppArmor profiles. Examples: runtime/default, unconfined."
               type: array
               items:
                 type: string
```

Comment:
Should we have a docstring for the individual item? Is that an option?

Reply:
Yes it is!  Would you prefer I put the examples there?

Reply:
Yep!